.pre-commit-config.yaml

```diff
@@ -1,6 +1,9 @@
+ci:
+    skip: [ansible-lint]
```

Comment:
Why are we both skipping the check and adding its dependencies?

Reply:
This is a similar change to PR https://github.com/Sage-Bionetworks-IT/packer-amazonlinux-docker/pull/17.  We actually don't need to skip this check for this specific repo because the ansible playbook in this repo doesn't use "community.general.sudoers".  I'm setting it up in this template repo in an attempt to make pre-commit for packer repos more resilient to errors.